extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/CertificateConfig.java

```diff
@@ -167,4 +171,15 @@ public class CertificateConfig {
      */
     @ConfigItem
     public Optional<String> trustStoreCertAlias;
+
+    /**
+     * When set, the configured certificate will be reloaded after the given period.
+     * Note that the certificate will be reloaded only if the file has been modified.
+     * <p>
+     * Also, the update can also occur when the TLS certificate is configured using paths (and not in-memory).
+     * <p>
+     * The reload period must be greater than 1 second.
+     */
+    @ConfigItem
+    public Optional<Duration> reloadPeriod;
```

Comment:
@cescoffier Does controlling reloading both the keystore and truststore files with the same property is flexible enough since the former is about letting this endpoint be trusted outside while the latter is about accepting the trusted client requests ?

I'd consider reloading keystores and truststores independently

Reply:
May be it is unnecessary...

Reply:
It's just the reload period; if there is a change in one, it will rebuild the SSL context, which contains both. So, it makes sense to have a single property for the server.

Note that this PR does not address the client. There is a bit more work (mostly because we have a ton of clients, in which we need to duplicate the same logic).